tests/opentelemetry/test_span.py

```diff
@@ -236,11 +234,11 @@ def test_otel_get_span_context_with_multiple_tracesates(oteltracer):
 
 
 def test_otel_get_span_context_with_default_trace_state(oteltracer):
-    otelspan = oteltracer.start_span("otel-server")
```

Comment:
We were never closing these spans before??

Reply:
Nope. It was contributing to some of the flakiness.